src/components/logo/logo.js

```diff
@@ -1,13 +1,24 @@
 import React from "react";
-
-import logo from "./assets/logo.png";
-
+import PropTypes from "prop-types";
 import "./logo.scss";
 
-export const Logo = () => {
+export const Logo = ({ src, url, alt, imgClass }) => {
   return (
     <div className="logo">
-      <img src={logo} alt="Code for Romania Task Force" />
+      {url ? (
+        <a href={url}>
```

Comment:
Can you please add target blank to these links?
```suggestion
        <a href={url} target="_blank" rel="noopener noreferrer">
```